#!/usr/bin/env python
PACKAGE = "optar"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("positionProcessVariance",			double_t,	0, "", 0.000001, 0.0000000001, 1000)
gen.add("positionMeasurementVariance",		double_t,	0, "", 10, 0.001,   1000)
gen.add("orientationProcessVariance",		double_t,	0, "", 0.001, 0.0000000001, 1000)
gen.add("orientationMeasurementVariance",	double_t,	0, "", 10, 0.0000000001, 1000)
gen.add("pnpMeasurementVarianceFactor",		double_t,	0, "", 1, 0.0000000001, 1000)
gen.add("arcoreMeasurementVarianceFactor",	double_t,	0, "", 20, 0.0000000001, 1000)

exit(gen.generate(PACKAGE, "optar", "OptarRegistrationEstimatorParameters"))
